ts/utils/keychain.ts

```diff
@@ -61,3 +62,14 @@ export async function getPin(): Promise<Option<PinString>> {
     return none;
   }
 }
+
+/**
+ * Saves the provided unlock code in the Keychain returning
+ * a `TaskEither` monad.
+ */
+export function safeSetPin(pin: PinString): TE.TaskEither<string, boolean> {
+  return TE.tryCatch(
+    () => setPin(pin),
+    e => `${e}`
```

Comment:
I'm not sure this is useful at all: the function eventually resolve to a [native binding](https://github.com/pagopa/io-app/pull/3939/files#diff-339040eb4b756ab97806623e8fd7080a8da4c9ab5ff8e7538996605f2d8e3c0eR28). The`error` might be a proper `Error` or something else. It may be worth checking against its type.

Reply:
Yes, this was already on my mind (that's why this PR is still a draft), at the end having a wrapper of a wrapper just for the `TaskEither` it's probably not worth it, I just switched back to the previous `setPin` function! 😺